nomad/structs/structs.go

```diff
@@ -859,6 +859,8 @@ type EvalDequeueRequest struct {
 type EvalListRequest struct {
 	FilterJobID      string
 	FilterEvalStatus string
+	FilterNamespace  string
+	OrderAscending   bool
```

Comment:
Does this need to go into the `api` package? I don't see `FilterJobID` or `FilterEvalStatus` either, but I think they were forgotten as well 😅